README.md

```diff
@@ -185,6 +185,12 @@ Default rules:
 
 - `no-print`: prevent usage of `print` statements in code (`error | warn | info | off`)
 
+- `newline-last`: enforces at least one newline (or absence thereof) at the end of non-empty files
```

Comment:
I think following ESlint's `eol-last` name would be sensible, but I'm ok with that